packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -184,4 +182,16 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 
 		return this.importedNotes[resolvedPath];
 	}
+
+	public isLinkToLocalFile(path: string) {
+		try {
+			const url = new URL(path);
+			if (url.protocol) return false;
+		} catch (error) {
+			// if it fails it probably is a relative path (local file)
+			if (error && error.code === 'ERR_INVALID_URL') return true;
+			throw error;
+		}
+		return false;
```

Comment:
I think your logic now is that if the protocol is defined you return false, and otherwise you return false. Basically I don't think your check on the protocol is doing anything

Reply:
By the way is the URL object defined on mobile?

Reply:
The idea was that if the link has a protocol it isn't a local file (`https:`, `mailto:`, `tel:`, `onenote:`, etc), so it would fail if we tried to instantiate the URL object.

I don't think it is an issue if the URL exists on mobile or not because this `Importer_Md` is one of the importer classes that are imported dynamically (because it isn't used by mobile anyways, we only use Importer/Exporter for Jex on mobile). Should I avoid using it anyway?

Reply:
> Should I avoid using it anyway?

I don't know, would you mind checking? Do we use `URL` elsewhere on mobile, how do we usually parse URLs on mobile?

> The idea was that if the link has a protocol it isn't a local file (https:, mailto:, tel:, onenote:, etc), so it would fail if we tried to instantiate the URL object.

That's not what your code does though - it instantiate URL and either returns true if it fails, or false otherwise. Protocol has no relevance to your code.